website/docs/docs/collaborate/dbt-explorer-faqs.md

```diff
@@ -63,6 +63,11 @@ dbt Explorer reflects all the lineage defined within the dbt project. Our vision
 
 </Expandable>
 
+<Expandable alt_header="Why does the dbt Explorer not show details that were there previously?" >
+
+The dbt Explorer will delete any stale data after 3 months so you want to ensure that you are running jobs with the necessary commands more frequently than that. 
```

Comment:
```suggestion
dbt Explorer automatically deletes stale metadata after 3 months if no jobs have been run to refresh it. To avoid this, make sure you schedule jobs to run more frequently than 3 months with the necessary commands.
```